<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>198</width>
        <height>400</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="LocalIP">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Local IP</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LIP">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="TCP">
         <property name="text">
          <string>TCP Packets</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="UDP">
         <property name="text">
          <string>UDP Packets</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="IP">
         <property name="text">
          <string>IP Packets</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ICMP">
         <property name="text">
          <string>ICMP Packets</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="StartButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Запустить Sniffer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ClearButton">
         <property name="text">
          <string>Очистить данные</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="StopButton">
         <property name="text">
          <string>Остановить Sniffer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="IPGenerator">
         <property name="text">
          <string>Сгенерировать IP пакет</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="SCRD">
         <property name="text">
          <string>SRC</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="SCRData">
         <property name="inputMask">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>255.255.255.255</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="DSTD">
         <property name="text">
          <string>DST</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="DSTData">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>255.255.255.255</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="GenerateButton">
         <property name="text">
          <string>Сгенерировать</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QTreeWidget" name="sniffs">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
